pkg/appregistry/appregistry.go

```diff
@@ -67,7 +67,18 @@ type AppregistryLoader struct {
 	loader     *dbLoader
 }
 
-func (a *AppregistryLoader) Load(csvSources []string, csvPackages string) (store registry.Query, err error) {
+func (a *AppregistryLoader) Load(csvSources []string, csvPackages string) (registry.Query, error) {
+	store, err := a.Load(csvSources, csvPackages)
```

Comment:
Should this be `a.load`?

Reply:
yup - I just pushed before I left the office